### DISCUSSION TITLE
Completed: Increase the character limit on Release pages.

### DISCUSSION THREAD

**Discussion by LlmDl** [community/community#4861] [Other Feature Feedback, Questions, & Ideas] [answered]
Sometime in the last 4 days there has been a new character limit imposed on the text boxes of the Release pages.

I'd like to ask for a higher limit or a return to when it was not limited (or the limit was so high it was unnoticeable.)

Just to show I'm hitting this sort of limit:
https://github.com/TownyAdvanced/Towny/releases/tag/0.97.0.17 

This pre-release still renders all of the lines, but if I attempt to edit it, I get the cut-off version. When it was time to put out the next pre-release (0.97.0.18) I ended up having to switch to using links to past pre-releases for many of the older pre-releases in this series of pre-releases.

**Comment by MylesBorins** [comment, marked as answer]
We had increased character limit a while back. Let me know if this is still an issue for you

**Reply by LlmDl** [reply]
I forgot to follow up on this, the issue did disappear. Thank you very much.
